src/manage_config_discovery.c

```diff
@@ -788,10 +788,10 @@ make_config_discovery (char *const uuid, char *const selector_name)
   NVT_SELECTOR (selector_name, "1.3.6.1.4.1.25623.1.0.80024", "Web Servers");
   NVT_SELECTOR (selector_name, "1.3.6.1.4.1.25623.1.0.19689", "Web Servers");
   NVT_SELECTOR (selector_name, "1.3.6.1.4.1.25623.1.0.10386", "Web Servers");
-  NVT_SELECTOR (selector_name, "1.3.6.1.4.1.25623.1.0.100075", "Useless services");
-  NVT_SELECTOR (selector_name, "1.3.6.1.4.1.25623.1.0.100080", "Useless services");
```

Comment:
The 100080 vuln reporting VT has been split into a separate detection reporting VT (should be available after the next feed sync):

https://secinfo.greenbone.net/omp?cmd=get_info&info_type=nvt&info_id=1.3.6.1.4.1.25623.1.0.108478&token=guest